install

```diff
@@ -125,8 +115,7 @@ do_install()
   rosinstall_generator ${ROS_CONFIGURATION} ${ROS_EXTRA_PACKAGES} --rosdistro ${ROS_DISTRO} --deps --tar > ${WS}.rosinstall
   wstool init -j8 src ${WS}.rosinstall
 
-  # Grabbing these older meshes allows rviz to run with Ogre 1.7 rather than Ogre 1.8+. Although formulae and patches
-  # exist to build Ogre 1.9, rviz, and Gazebo together, I have found this approach to be the most solid and consistent.
+  # Grabbing these older meshes allows rviz to run with Ogre 1.7 rather than Ogre 1.8+.
   if [ -d src/rviz ]; then
     pushd src/rviz/ogre_media/models
     curl https://raw.githubusercontent.com/ros-visualization/rviz/hydro-devel/ogre_media/models/rviz_cone.mesh > rviz_cone.mesh
```

Comment:
The real problem with Ogre 1.9 was Gazebo IIRC, although the last time I checked was with gz5, so things may well have changed since then. That said, the fact that `osrf/simulation/ogre` still installs 1.7.4 does seem like a vote of non-confidence, though maybe yourself or @scpeters can speak more to that.

In any case, I don't have the will or energy to maintain my own homebrew formulas; this installer is absolutely meant to guide users down the path of least resistance, whatever that happens to be at a given moment in time.